Smooch.postman_collection.json

```diff
@@ -2054,7 +2054,7 @@
 						],
 						"body": {
 							"mode": "raw",
-							"raw": "{\n\t\"type\": \"twilio\",\n\t\"confirmation\": {\n\t\t\"type\": \"prompt\",\n\t\t\"phoneNumber\": \"+15145555555\"\n\t}\n}"
+							"raw": "{\n\t\"type\": \"mailgun\",\n\t\"confirmation\": {\n\t\t\"type\": \"immediate\"\n\t},\n\t\"address\": \"their@email.com\"\n}"
```

Comment:
Dunno where this shows up in postman, but there's also a badly formatted request body on line 2088

Reply:
![image](https://user-images.githubusercontent.com/54906681/82825476-4f7c5080-9e79-11ea-8345-627a23e6448f.png)
IUf you click `Examples` in the top right corner, it'll show the request/response example. Twilio is included there in the body of both.

![image](https://user-images.githubusercontent.com/54906681/82825551-72a70000-9e79-11ea-8a77-54ef874c8630.png)
Also Status: `404 Not Found` should be changed to 200.

Reply:
Thanks, I didn't know about Examples!